src/jdk.incubator.foreign/share/classes/jdk/incubator/foreign/CLinker.java

```diff
@@ -114,6 +111,13 @@
  */
 public interface CLinker {
 
+    /**
+     * The value returned by the process when the target of an upcall throws an exception.
+     *
+     * @see CLinker#upcallStub(MethodHandle, FunctionDescriptor, ResourceScope)
+     */
+    int ERR_UNCAUGHT_EXCEPTION = 1;
```

Comment:
On a second thought, I'm not super sure of the value of documenting what the exit code should be? Maybe we can leave that unspecified?

Reply:
I agree. Without a system property, having a constant doesn't really make sense. I looked around in the documentation of exit codes in the JDK, and most  places seem to at least specify the exit code  to be non-zero in case of error. I'll add a note like that to the documentation of upcallStub.